simglucose/simulation/env.py

```diff
@@ -34,7 +36,58 @@ def risk_diff(BG_last_hour):
 
 
 class T1DSimEnv(object):
+    '''
+    Observation:
+        Type: Box(4)
+        Num     Observation                 Min         Max
+        0       BG                          69.         351.
+        1       Insulin                     0.0         Inf
+        2       CHO                         0.0         Inf
+        3       CGM                         0.0         Inf
+
+    Actions:
+        Type: Discrete(2)
+        Num     Action
+        0       Adjust Insulin Bolus Down
```

Comment:
In this `simglucose.simulation.env.T1DSimEnv` class, the action space is continuous `Box(2)`. The first action is insulin basal value, and the second action is insulin bolus value. In the gym environment `simglucose.envs.simglucose_gym_env.T1DSimEnv`, the action space is `Box(1)`, and only insulin basal value is controllable. Ideally, the agent should do a decent job by only controlling the basal value.